blazor/gantt-chart/performance.md

```diff
@@ -0,0 +1,101 @@
+* [Injecting service into CustomAdaptor](https://blazor.syncfusion.com/documentation/gantt-chart/custom-binding#inject-service-into-custom-adaptor)
+
+## How to improve loading performance by referring individual script and CSS
+
+Instead of using the consolidated `Syncfusion.Blazor` package, which includes all Syncfusion Blazor components and can result in a larger package size, you should use the specific NuGet package for the Gantt Chart component, such as `Syncfusion.Blazor.Gantt`. This approach ensures that only the necessary components and dependencies for the Gantt Chart are included, leading to improved performance and reduced load times.
+
+When configuring your Blazor `Gantt Chart`, use individual script and CSS files instead of a single large package file. This practice allows for more efficient loading and rendering of the component by reducing the amount of data processed during initialization. By referring to individual scripts and CSS files, you can ensure that only the necessary resources are loaded, which enhances performance and speeds up the initial rendering of the Gantt Chart.
+
+Refer the below documentation
+* [Individual nuget package](https://blazor.syncfusion.com/documentation/gantt-chart/getting-started-with-web-app#install-syncfusion-blazor-gantt-and-themes-nuget-in-the-blazor-web-app)
+* [Adding script and CSS](https://blazor.syncfusion.com/documentation/gantt-chart/getting-started-with-web-app#add-stylesheet-and-script-resources)
+
+By adopting these strategies, you can significantly improve the performance of the Blazor Gantt Chart during initial rendering and ensure a more responsive user experience.
+
+## How to optimize server-side data operations with adaptors
+
+The Blazor Gantt Chart supports various adaptors (OData, ODataV4, WebAPI, URL, etc.) to facilitate server-side data operations and CRUD functionalities. By utilizing these adaptors along with the [SfDataManager](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Data.SfDataManager.html) component, you can seamlessly bind remote data sources to the Gantt Chart and execute actions. During data operations such as filtering and sorting, the corresponding action queries are generated according to the adaptor’s requirements. It is crucial to handle these actions on the application side and return the processed data back to the Gantt Chart. For efficient data processing, the suggested order for returning processed data to the Gantt Chart is as follows:
+
+* Filtering
+* Sorting
+
+## Strategic approaches to addressing latency challenges
+
+Understanding the concerns related to latency in the Syncfusion Blazor Gantt Chart component, several factors contributing to responsiveness issues have been identified. Notably, when using features like filtering, sorting, and data binding in the Gantt Chart, delays may occur due to client-server interactions, especially if the server is located far from the client.
```

Comment:
remove sorting feature and data binding. And add taskbar resizing, and dialog edit feature.

remove the word especially